### PR TITLE
Add axiomatic 'define' tests

### PR DESCRIPTION
**Pull request by alexjpwalker**
This PR forms part of the work towards resolving:

Complete Axiomatic Tests - implement missing Graql query scenarios #18

It adds a large number of 'define' tests, as well as a new test step 'graql define throws' to be implemented in client-java and grakn.

### CODE REVIEW

File: behaviour/graql/language/define.feature [hunk 1]
```diff
@@ -338,66 +744,352 @@ Feature: Graql Define Query
...
+
+
+  Scenario: define attribute type throws if 'value' is invalid
+    Then graql define throws
+      """
+      define colour sub attribute, value rgba;
+      """
+
+
+  @ignore
```
Comment (flyingsilverfin) [inline behaviour/graql/language/define.feature:824]: make this into an issue in `verification` so we don't for get :)

File: behaviour/graql/language/define.feature [hunk 2]
```diff
@@ -338,66 +744,352 @@ Feature: Graql Define Query
...
+      |     | check | value                      |
+      | NOL | label | number-of-limbs            |
+      | NAL | label | number-of-artificial-limbs |
+    Then uniquely identify answer concepts
+      | x   |
+      | NOL |
+      | NAL |
+
+
+  Scenario: define a type as abstract errors if has non-abstract parent types (?)
```
Comment (flyingsilverfin) [inline behaviour/graql/language/define.feature:1085]: this meant to be empty? Along with the ones below?

File: behaviour/graql/language/define.feature (hunk 1 above)
Comment (alexjpwalker) [reply behaviour/graql/language/define.feature:824]: https://github.com/graknlabs/verification/issues/46

File: behaviour/graql/language/define.feature (hunk 2 above)
Comment (alexjpwalker) [reply behaviour/graql/language/define.feature:1085]: For now, yes. I intend to implement them once #46 is resolved.

**Pull request merged**